src/container/profile/EditProfile.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Button, Icon } from 'antd';
+import EditProfileForm from '../../components/profile/forms/EditProfileForm';
+
+import { editUserProfile } from '../../actions/profile';
+
+export class UnconnectedEditProfile extends React.Component {
+  state = {
+    visible: false,
+  };
+
+  showModal = () => {
+    this.setState({ visible: true });
```

Comment:
You seem to be accessing the state directly. In redux, the store is the single source of truth. The state can only be updated through the store. Instead of updating the state directly, consider dispatching an action which then will call a reducer to update the state.

Reply:
Don't you think that would only make things complicated considering no other component is affected by this and it only toggles true and forth. It seems much more readable here that having to call an action every time.

Reply:
It then beats the purpose of using Redux to manage state, in my opinion, of course, if we are going to update the state directly.  Don't you think?

Reply:
I have made the correction. I realised there was already an action for that so my work was just importing. Thanks for the review.

Reply:
LGTM